aspnetcore/mvc/razor-pages/ui-class/samples/cli/RazorUIClassLib/RazorUIClassLib.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.1.0-rc1-30661" />
```

Comment:
`2.1.0-rc1-final` or just `2.1.0`. Think we're close to getting RTM bits any day now